server/bleep/migrations/20230919100529_code_studio_docs.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE docs (
+    id INTEGER PRIMARY KEY,
+
+    url TEXT NOT NULL,
+    index_status TEXT NOT NULL,
+
+    name TEXT,
+    favicon TEXT,
+    description TEXT,
```

Comment:
Can these also be `NOT NULL`?

Reply:
we won't always be able to scrape metadata from a page.